lib/rspec/abq/formatter.rb

```diff
@@ -82,6 +87,27 @@ def self.abq_result(example)
           end
         end
       end
+
+      private_class_method def self.rspec_retry_attempts(example)
+        return unless defined?(RSpec::Retry)
+        return unless example.metadata.key?(:retry_attempts)
+        return unless example.metadata.key?(:retry_exceptions)
+
+        retry_attempts = example.metadata[:retry_attempts]
+        retry_exceptions = example.metadata[:retry_exceptions]
+        return unless retry_attempts > 0
+
+        retry_attempts.times.map do |attempt_index|
+          {
+            status: {type: :failure},
+            id: example.id,
+            display_name: example.metadata[:full_description],
+            output: retry_exceptions[attempt_index],
+            runtime: 0, # unknown
+            meta: {}
+          }
+        end
```

Comment:
Unfortunately the ABQ protocol requires some of this because it calls past attempts just another test result spec. In the RWX Test Results schema, much of this is not necessary for a past attempt (really just the status)

Reply:
And ultimately much of it is dropped, though the 0 duration is unfortunate